### ISSUE TITLE
QoL feature list

### ISSUE THREAD

**Issue by 787992** [GaijinEntertainment/fmod_studio_warthunder_for_modders#183]
Hey again, today just dropped update for autoloaders and two sound engine event fix.
I love that, but you could you please also take a look at [https://github.com/GaijinEntertainment/fmod_studio_warthunder_for_modders/issues/142](Suggestions and fixes topic) At least address the issues with reload and cannon firing sounds that cannot be in 3d and broken explosions_rapid sound events, that works only on map props, but ignores the terrain surface.

**Comment by Uchuujin01** [comment]
Hello! I do remember about it, thanks again. But now I have no spare time for it ( Hope step by step I process your list with time.

**Comment by Hosiokaa** [comment]
Indeed. Gaijin please look into this.

**Comment by Odican** [comment]
I agree, Gaijin please don't ignore this

**Comment by FenixAlfa159** [comment]
This would be great! I look forward to this!

**Comment by NorthyPark** [comment]
Yeah, please gaijin take a look at this, since getting those fixed would be great!

**Comment by IWSM** [comment]
check

**Comment by Gentlespie** [comment]
This would be a nice thing honestly, please bring it up to the rest of devs and programmers!

**Comment by IWSM** [comment]
> Hello! A part of your suggestions is under consideration. We like some of them. I can't promise we implement all of them, but definitely something should be implemented, like diversity of aircraft rockets. Thank you for helping us!

Thx you to

**Comment by 787992** [comment]
> Hello! A part of your suggestions is under consideration. We like some of them. I can't promise we implement all of them, but definitely something should be implemented, like diversity of aircraft rockets. Thank you for helping us!

Good to know! My also high hopes on addressing long awaited issue with delayed sound of reloading and firing the cannon.
I bet a lot of sound modders wanted to utilize the feature of making different exterior and interior reload noises, especially after seeing corresponded sound samples in official project files.
